DOL.WHD.Section14c.Web/src/modules/components/sectionEmployer/sectionEmployerController.test.js

```diff
@@ -26,6 +26,25 @@ describe('sectionEmployerController', function() {
 
     expect(scope.formData.employer.tradeName).toBe('');
   });
+
+  it('has n/a for country when DC is selected as state', function() {
+    var controller = sectionEmployerController();
+
+    scope.formData.employer.physicalAddress = {};
+    scope.formData.employer.physicalAddress.state = 'DC';
+    scope.$digest();
+    expect(scope.formData.employer.physicalAddress.county).toBe('N/A');
+  });
+
+  it('has n/a for country when DC is selected as state', function() {
```

Comment:
I think this test name needs to be updated since it is checking for a county when the state is Pennsylvania?